server/scripts/2022TaxSeasonDataScript.php

```diff
@@ -445,29 +445,29 @@ function insertInternationalStudentScholarSiteData() {
 		$tuesdayDates = getWeeklyDatesFromRange('2022-03-01', '2022-04-05');
 		foreach ($tuesdayDates as $date) {
 
-			// Virtual China
```

Comment:
Wasn't this script run already? You'll have to update these manually via the management page if so--but I still like this update for posterity

Reply:
tldr; this is just for posterity!

You're right that this was run already, I actually ran the script again with only ISS to supplement the already scheduled in-person appointments--so if 3/10 appointments had been scheduled, I used the management page to drop it to 3/3, then changed the script on the server to add 7 in-person appointments to restore the total to the original 10. It wasn't ideal, but it was time-sensitive and I thought this was safer (and maybe faster?) than trying to run and test a script during live tax season. Thanks as always for checking--you never know when I might have misunderstood something.